src/hooks/useLocalhostQuickstart.js

```diff
@@ -0,0 +1,39 @@
+import { useEffect, useState } from 'react';
+
+import { getQuickstartFilesFromLocal } from '../utils/preview/fetchHelpers';
+import { parseQuickstartFiles } from '../utils/preview/parseHelpers';
+import { navigate } from 'gatsby';
+
+const usePullRequestQuickstart = (location) => {
```

Comment:
I think this is just a copy paste error, it's got the wrong hook name.
```suggestion
const useLocalhostQuickstart = (location) => {
```

Reply:
Ahh good catch.  I just opened a new PR which doesnt have Tabitha's commit to keep it separate,  I'm going to update it on this one:
https://github.com/newrelic/instant-observability-website/pull/193